src/shared-components/button/shared-components/loader/style.js

```diff
@@ -35,7 +35,7 @@ const ButtonLoader = styled.div`
   right: 0;
   top: 0;
   margin-top: -6px;
-  width: 38px;
+  width: ${({ isFullWidth }) => (isFullWidth ? `25%` : `38px`)};
```

Comment:
25% was fairly arbitrary but was relatively close for very wide buttons (which isn't a pattern we want to use, since we're adding this for mobile), but also close enough with narrower viewports (320-400px).